CMakeLists.txt

```diff
@@ -129,6 +129,12 @@ target_include_directories(app_lib PUBLIC
         ${CMAKE_CURRENT_SOURCE_DIR}/app/src/common
         )
 
+string(APPEND CMAKE_C_FLAGS   " -include /usr/include/bsd/string.h")
+string(APPEND CMAKE_CXX_FLAGS " -include /usr/include/bsd/string.h")
```

Comment:
I doubt these changes are necessary.

Reply:
I couldn't compile the project without them on an archlinux and on a ubuntu:20.04 container. I just tried on the a github workflow when reverting the commit and I got the same behavior (cf [pipeline](https://github.com/qbLedger/app-edgeware/runs/4819333056?check_suite_focus=true)). Notice this *CMakeFile.txt* does not build the actual release using Ledger's toolchain but compiles unit test for the project, using the system libraries.